drake/addpath_drake.m

```diff
@@ -60,12 +60,12 @@
 javaaddpath(fullfile(pods_get_base_path,'share','java','drake.jar'));
 
 % OSX platform-specific: check if reverted to IPv4
-if (computer('arch') == 'maci64')
+if (strcmp(computer('arch'),'maci64'))
   ipv4_preferred = java.lang.System.getProperty('java.net.preferIPv4Stack');
   if isempty(ipv4_preferred)
     ipv4_preferred = 'false';
   end
-  if (ipv4_preferred ~= 'true')
+  if (strcmp(ipv4_preferred,'true'))
```

Comment:
This fix is backwards; the previous version checked if ipv4_preferred was *not* equal to `'true'`, but this checks if it *is* equal